src/main/java/org/openrewrite/java/testing/junit5/UpdateTestAnnotation.java

```diff
@@ -149,6 +150,7 @@ public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration method, Ex
                                 .staticImports("org.junit.jupiter.api.Assertions.assertThrows")
                                 .build()
                                 .apply(updateCursor(m), m.getCoordinates().replaceBody(), cta.expectedException, lambda);
+                        m = (J.MethodDeclaration) new UnnecessaryThrows().getVisitor().visit(m, ctx, getCursor().getParentOrThrow());
```

Comment:
As indicated; quite likely that we can not use `UnnecessaryThrows` directly, since that limits itself to avoid doing harm. We could create a separate visitor in rewrite-testing-frameworks that's a little more aggressive as we have a smaller scope to work with.